pkgs/applications/graphics/inkscape/test-ps2pdf-plugin.nix

```diff
@@ -0,0 +1,58 @@
+{ inkscape, runCommand, writeTextFile }:
+
+let
+  svg_file = writeTextFile {
```

Comment:
Maybe we could move this SVG into a separate file or use an existing svg file like in

https://github.com/NixOS/nixpkgs/blob/d97cd093db0726e261a54308fd0c3f71c01bfcbe/pkgs/development/libraries/xdg-desktop-portal/default.nix#L132

Reply:
hello @jtojnar , 
I tried to reduce the svg file to a minimum, but would keep external deps to a minimum.
So our test gets not broken by a change in another package.

If you think otherwise I will change to the external file.